README.md

```diff
@@ -8,16 +8,165 @@ Local.
 For more information, see [Getting started with Cloudant NoSQL DB](https://www.ng.bluemix.net/docs/services/Cloudant/index.html)
 
 The `loopback-connector-cloudant` module is the Cloudant connector for the Loopback framework.
+<!-- TOC -->
+
+- [loopback-connector-cloudant](#loopback-connector-cloudant)
```

Comment:
I mean we should not have that there at all. [This](https://github.com/strongloop/loopback-connector-cloudant/blob/refactor/doc/README.md#loopback-connector-cloudant) already points me to the `loopback-connector-cloudant`. Small thing, if you want to leave as it is I am okay :)